### PR TITLE
#298/hoover states and special effects/2.1

### PR DESCRIPTION
**Pull request by pierwszazlewej**
- subtle hover state 
obraz

### CODE REVIEW

File: client/src/Atoms/ButtonsExercises/ButtonExercisesStyled.js [hunk 1]
```diff
@@ -19,6 +19,12 @@ const ButtonExercisesStyled = styled.button`
   margin: 0.25rem 0;
   ${FONTS.label_semibold};
   ${(props) => handleBgAndColor(props)};
+
+  &:hover {
+    filter: brightness(1.05);
+    box-shadow: 2px 4px ${({ theme }) => theme.primary201};
```
Comment (n0macx) [inline client/src/Atoms/ButtonsExercises/ButtonExercisesStyled.js:25]: Maybe we should stick to rem values ?

File: client/src/Consts/rolesConsts.js [hunk 2]
```diff
@@ -4,11 +4,11 @@ export const ROLES = {
     roleButtonLink: '/user',
   },
   staff: {
-    roleButtonText: 'Konto Obsługi',
+    roleButtonText: 'Panel sędziego',
```
Comment (n0macx) [inline client/src/Consts/rolesConsts.js:7]: Capital Judge ?

File: client/src/Atoms/ButtonsExercises/ButtonExercisesStyled.js (hunk 1 above)
Comment (pierwszazlewej) [reply client/src/Atoms/ButtonsExercises/ButtonExercisesStyled.js:25]: fixed

File: client/src/Consts/rolesConsts.js (hunk 2 above)
Comment (pierwszazlewej) [reply client/src/Consts/rolesConsts.js:7]: fixed

**Review by Smyku6 — APPROVED** [review]

**Pull request merged**